src/hotspot/share/classfile/classFileParser.cpp

```diff
@@ -3919,8 +3913,8 @@ void ClassFileParser::parse_classfile_attributes(const ClassFileStream* const cf
             parsed_record_attribute = true;
             record_attribute_start = cfs->current();
             record_attribute_length = attribute_length;
-          } else if (tag == vmSymbols::tag_permitted_subclasses()) {
-            if (supports_sealed_types()) {
+          } else if (_major_version >= JAVA_17_VERSION) {
```

Comment:
Can you update to the comment at L3932 to say JAVA_17_VERSION please.

Reply:
sure